elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -1707,4 +1685,27 @@ public static String getModelVersion(Class<?> modelClass) {
 
         return (apiVersionAnnotation == null) ? NO_VERSION : apiVersionAnnotation.version();
     }
+
+    /**
+     * Looks up the API model name for a given class.
+     * @param modelClass The model class to lookup.
+     * @return the API name for the model class.
+     */
+    public static String getEntityName(Class<?> modelClass) {
+        Class<?> declaringClass = lookupAnnotationDeclarationClass(modelClass, Include.class);
```

Comment:
@aklish This caused a bug because Include is not necessarily on an Entity bean, but can be instead in the package-info.